espnet/tts/pytorch_backend/tts.py

```diff
@@ -223,23 +221,21 @@ def train(args):
         logging.info('ARGS: ' + key + ': ' + str(vars(args)[key]))
 
     # specify model architecture
-    tacotron2 = Tacotron2(idim, odim, args)
-    logging.info(tacotron2)
+    model_class = dynamic_import(args.model_module)
```

Comment:
Assert what you expect the `model_class` to implement whenever you use `dynamic_import` because pycharm or some static analysis tool support that. I'm planning to update the `CONTRIBUTING.md`.

See https://github.com/espnet/espnet/blob/a8e18ac15396986143404e4f339c818a5a57bdc2/espnet/asr/pytorch_backend/asr.py#L255

Reply:
I mean `assert isinstance(model, TTSInterface) ` here